sdk/identity/identity/test/internal/node/onBehalfOfCredential.spec.ts

```diff
@@ -29,7 +29,7 @@ describe("OnBehalfOfCredential", function () {
       userAssertionToken: "user-assertion",
     });
 
-    const newMSALClientLogs = () =>
+    const newMSALClientLogs = (): number =>
```

Comment:
This is just for clarity right? was it causing some issue? curious to know @mpodwysocki

Reply:
maybe a lint thing, it's common to enforce explicitly declaring return types to avoid accidentally returning the wrong thing